ott/log_parser/db/processed_requests.py

```diff
@@ -35,59 +40,176 @@ class ProcessedRequests(Base):
         lazy="joined", innerjoin=True,
     )
 
+    related = relationship(
```

Comment:
related not used / populated atm
should probably be related to ProcessedReq id rather than RawLog id